### PR TITLE
clarification on LTS support

### PR DESCRIPTION
**Pull request by mcollina**
@fastify/fastify please check this out, I've added some more wording to clarify what will be expected from us for v1.

### CODE REVIEW

File: docs/LTS.md [hunk 1]
```diff
@@ -13,7 +13,15 @@ out in this document:
    [https://github.com/fastify/fastify/releases](https://github.com/fastify/fastify/releases).
 
 1. Major releases will receive security updates for an additional six months
-   from the release of the next major release.
+   from the release of the next major release. After this period is expired,
+   we will still review and release security fixes as long as they are
+   provided by the community and they do not violate other constrains,
+   e.g. minimum supported Node.js version.
+
+1. Major release will be tested and verified against all Node.js
```
Comment (cemremengu) [inline docs/LTS.md]: Major release`s`

File: docs/LTS.md [hunk 2]
```diff
@@ -13,7 +13,15 @@ out in this document:
...
+   releases that are supported by the
+   [Node.js LTS policy](https://github.com/nodejs/Release) within the
+   LTS period of that given Fastify release line.
```
Comment (cemremengu) [inline docs/LTS.md]: LTS period of `a` Fastify release line

(sounds simpler I think)

File: docs/LTS.md [hunk 3]
```diff
@@ -13,7 +13,15 @@ out in this document:
    [https://github.com/fastify/fastify/releases](https://github.com/fastify/fastify/releases).
 
 1. Major releases will receive security updates for an additional six months
-   from the release of the next major release.
+   from the release of the next major release. After this period is expired,
+   we will still review and release security fixes as long as they are
+   provided by the community and they do not violate other constrains,
```
Comment (jsumners) [inline docs/LTS.md]: "contrains" => "contraints"

File: docs/LTS.md (hunk 2 above)
Comment (jsumners) [reply docs/LTS.md]: The "given" clause is necessary to indicate it is specific to the LTS Fastify release.

File: docs/LTS.md (hunk 3 above)
Comment (nwoltman) [reply docs/LTS.md]: Whoops, looks like you also have a typo @jsumners
"contraints" => "constraints"

Comment (jsumners) [reply docs/LTS.md]: 🤣

File: docs/LTS.md [hunk 4]
```diff
@@ -23,18 +31,18 @@ A "month" is to be a period of 30 consecutive days.
 
 ### Schedule
 
-| Version | Release Date | End Of LTS Date |
-| :------ | :----------- | :-------------- |
-| 1.0.0   | 2018-03-06   | TBD             |
+| Version | Release Date | End Of LTS Date | Node.js         |
+| :------ | :----------- | :-------------- | :-------------- |
+| 1.0.0   | 2018-03-06   | TBD             | 6, 8, 9, 10, 11 |
```
Comment (delvedor) [inline docs/LTS.md]: `End Of LTS Date` => `2019-06-01`?

Comment (delvedor) [reply docs/LTS.md]: We should also add v2.

**Pull request merged**